guide/getting-started-nodejs.md

```diff
@@ -0,0 +1,139 @@
+## Getting started with NodeJS on OpenFaaS
+
+There are a number of resources that document getting started with
+OpenFaaS, in fact the entire stack can be deployed in less than 60
+seconds when following the instructions from the README on GitHub!
+This article will focus on how easy it is to setup a simple NodeJS
+function and deploy it.
+
+#### Dependencies
+
+This article assumes installation of at least `Docker 17.05`. Docker
```

Comment:
Let's scrap this paragraph and say that we're using "Docker Swarm" with Docker 17.05 or later.

Reply:
Still needs to mention Docker Swarm..